verticapy/utilities.py

```diff
@@ -485,19 +483,27 @@ def pandas_to_vertica(
 read_csv  : Ingests a CSV file into the Vertica database.
 read_json : Ingests a JSON file into the Vertica database.
 	"""
+    check_types(
+        [
+            ("name", name, [str],),
+            ("schema", schema, [str],),
+        ]
+    )
     cursor, conn = check_cursor(cursor)[0:2]
-    name = "verticapy_df_{}".format(random.randint(10e5, 10e6))
+    if not(name):
+        name = "verticapy_df_{}".format(random.randint(10e5, 10e6))
     path = "{}.csv".format(name)
     try:
         df.to_csv(path, index=False)
-        read_csv(path, cursor, table_name=name, temporary_local_table=True, parse_n_lines=10000)
+        if not(name):
```

Comment:
You have added `if not(name):` a few lines above to treat `name`. Then, this condition will never be satisfied.

Reply:
Corrected;